Adding SLOs troubleshooting document (attempt 2) (#4486)
elastic/observability-docs: docs/en/observability/slo-create.asciidoc

. <<set-slo>>.
. <<slo-describe>>.

[NOTE]
====

> For SLOs to function, the cluster must include one or more nodes with both `ingest` and `transform` {ref}/modules-node.html#node-roles[roles] (they can co-exist or be distributed across separate nodes).

Reviewer: ```suggestion
For SLOs to function, the cluster must have one or more nodes with both `ingest` and `transform` {ref}/modules-node.html#node-roles[roles]. The roles can exist on the same node or be distributed across separate nodes.
```

Reviewer: Sorry missed one...I've tweaked the text in the other statement to match this one.
